### PR TITLE
Something to start with

### PR DESCRIPTION
**Pull request by wiiitek**
I had some issues related to finding bundle jar file on windows...

While debugging I found that the stream for decompilation is null. Then that the jar file is null. So I added a lot of debug messages in order to find the source of this. I know that they are not too pretty...

I was using this tool with only felix framework 5.6.2 + some additional bundles for WebConsole and logging support. This may be not supported use-case... :)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
     </developers>
 
     <build>
-        <resources>
```
Comment (wiiitek) [inline pom.xml:59]: This resources declaration looks like could be removed. Or am I wrong?

File: src/main/java/com/neva/felix/webconsole/plugins/search/core/OsgiExplorer.java [hunk 2]
```diff
@@ -61,28 +62,50 @@ public OsgiExplorer(BundleContext context) {
 	}
 
 	public File findDir(Long bundleId) {
-		return new File(String.format(BUNDLE_PATH_FORMAT, context.getProperty(BUNDLE_STORAGE_PROP), bundleId));
+		String bundleStorage = context.getProperty(BUNDLE_STORAGE_PROP);
```
Comment (wiiitek) [inline src/main/java/com/neva/felix/webconsole/plugins/search/core/OsgiExplorer.java:65]: This line returns the config value from felix configuration.
In my `config.properties` file i have an entry of: `${felix.cache.rootdir}/felix-cache/bundles`.
Then `bundleStorage == "/felix-cache/bundles"` and i think that on windows this is treated as absolute path.
When I don't have this property configured then felix uses default folder: `felix-cache`, but then the property is null.

At the end i have used the configuration of:
```
org.osgi.framework.storage=./felix-cache/bundles
```
but something is still wrong :?

Comment (pun-ky) [reply src/main/java/com/neva/felix/webconsole/plugins/search/core/OsgiExplorer.java:65]: Leading slash is just wrong configuration. On what type of osgi container you noticed problems? Maybe I can reproduce problem. I tested plugin on both windows and ubuntu. Maybe leading dot and slash need to be handled separately.. I will look at this. About PR . I don't want to put all of this logging entries. In my opinion they are messy...

Comment (pun-ky) [reply src/main/java/com/neva/felix/webconsole/plugins/search/core/OsgiExplorer.java:65]: If you used default config of clean Felix.. This must be covered. I will look at it today

Comment (pun-ky) [reply src/main/java/com/neva/felix/webconsole/plugins/search/core/OsgiExplorer.java:65]: maybe please provide a felix framework zipped you used...

Comment (pun-ky) [reply src/main/java/com/neva/felix/webconsole/plugins/search/core/OsgiExplorer.java:65]: I checked that not only empty property on clean felix is a problem.
I will put just "./felix-cache" as default value if prop not specified. One problem solved.
But there is some class loading issue when decompile servlet is called.

```
11:36:36.300 [qtp1456240109-30] WARN org.eclipse.jetty.servlet.ServletHandler - Error for /system/console/search/class-decompile
java.lang.NoClassDefFoundError: Could not initialize class com.strobel.assembler.metadata.MetadataSystem
        at com.strobel.decompiler.Decompiler.decompile(Decompiler.java:34)
        at com.neva.felix.webconsole.plugins.search.core.OsgiExplorer.decompileClass(OsgiExplorer.java:130)
        at com.neva.felix.webconsole.plugins.search.rest.ClassDecompileServlet.doGet(ClassDecompileServlet.java:64)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:618)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
        at org.apache.felix.http.base.internal.handler.ServletHandler.hand
```

To be investigated...

**Pull request closed without merging**